src/mem3/src/mem3.erl

```diff
@@ -423,31 +424,127 @@ engine(Opts) when is_list(Opts) ->
 %% Check whether a node is up or down
 %%  side effect: set up a connection to Node if there not yet is one.
 
--spec ping(Node :: atom()) -> pong | pang.
+-spec ping(Node :: atom()) -> pos_integer() | Error :: term().
```

Comment:
I couldn't find any usage or in the API or docs about it so figured `pang` is less useful than an explanation why it failed, and `pong` is less useful than an actual time it took to respond. Then it would also be somewhat compatible with ping_nodes.